MANIFEST.in

```diff
@@ -24,7 +25,9 @@ include pyproject.toml
 include README.rst
 include requirements/requirements*.txt
 include tox.ini
+recursive-include conf *.conf
```

Comment:
🤘

Reply:
The sdist should only include source files, not configs used for dev testing, I think.